src/PHPQueue/Backend/Predis.php

```diff
@@ -6,7 +6,19 @@
 use PHPQueue\Interfaces\FifoQueueStore;
```

Comment:
Good point about "score", I'm hiding that detail in favor of "order key" which should be more descriptive.